sherpa/models/basic.py

```diff
@@ -40,14 +40,91 @@
 __all__ = ('Box1D', 'Const1D', 'Cos', 'Delta1D', 'Erf', 'Erfc', 'Exp', 'Exp10',
            'Gauss1D', 'Log', 'Log10', 'LogParabola', 'NormGauss1D', 'Poisson',
            'Polynom1D', 'PowLaw1D', 'Scale1D', 'Sin', 'Sqrt', 'StepHi1D',
-           'StepLo1D', 'Tan', 'Box2D', 'Const2D', 'Delta2D', 'Gauss2D',
-           'SigmaGauss2D',
+           'StepLo1D', 'Tan', 'Voigt1D',
+           'Box2D', 'Const2D', 'Delta2D', 'Gauss2D', 'SigmaGauss2D',
            'NormGauss2D', 'Polynom2D', 'Scale2D', 'UserModel', 'TableModel',
            'Integrate1D')
 
 DBL_EPSILON = numpy.finfo(numpy.float).eps
 
 
+class Voigt1D(RegriddableModel1D):
+    """One dimensional Voigt profile.
+
+    The Voigt profile is a convolution between a Gaussian distribution
+    a Cauchy-Lorentz distribution [1]_, [2]_.
+
+    .. versionadded:: 4.12.2
+
+    Attributes
+    ----------
+    fwhm_g
+        The full-width half-maximum (FWHM) of the Gaussian distribution.
+    fwhm_l
+        The full-width half-maximum of the Lorentzian distribution.
+    pos
+        The center of the profile.
+    ampl
+        The amplitude of the profile..
```

Comment:
I guess the amplitude is defined as an integral of the profile and equivalent of the EW - equivalent width. 
In our old documentation we give a bit more details about the profile = the core is described by a Gaussian and the wings by a Lorentzian.

Reply:
I am not 100% convinced the old description was reliable.

I have no idea whether the amplitude is the same as the EW or not. You'll have to look down to where I give the formula describing the model, and see.

I am reluctant to add significantly more text here, as I have provided references for people who want to find out more, and the more text we have, the more we have to update when we want to change things.

Reply:
`lmfit` output gives the ampl and height. I assume that ampl is the integral of the profile, but I have not written the code, so I don't know how it is calculated.  However, the calculation/definition of `ampl` needs to be explained otherwise it is ambiguous parameter.

Reply:
There's a notes section below that describes exactly how ampl and the other parameters are used/defined - it ends at https://github.com/sherpa/sherpa/pull/770/files#diff-dc85eaecd2ce7d045722ea9aa44fab66R93

Reply:
Yes, it refers to Faddeeva function...

Reply:
which is referenced so you can find out what it is.